test/integration/users.test.js

```diff
@@ -1503,6 +1503,27 @@ describe("Users", function () {
     });
   });
 
+  describe("PATCH /update-archived", function () {
+    it("returns successful response", function (done) {
+      chai
+        .request(app)
+        .patch("/users/update-archived")
+        .set("cookie", `${cookieName}=${superUserAuthToken}`)
+        .end((err, res) => {
+          if (err) {
+            return done(err);
+          }
+
+          expect(res).to.have.status(200);
+          expect(res.body.data).to.have.property("totalUsersWithArchivedRoleUpdated");
```

Comment:
can we also assert `totalUsersWithArchivedRoleUpdated ` value as well to check >0 and actually working